public/js/gulliver.es6.js

```diff
@@ -129,6 +129,104 @@ function setupSignedinAware() {
   }
 }
 
+function fetchInnerContent(element, newUrl) {
```

Comment:
We should make this code more OO, and move it to it's own class, on a separate file, and then import from gulliver.js.